src/types/User.ts

```diff
@@ -39,7 +39,12 @@ export interface User {
   myPersonalizedSchedule?: MyPersonalizedSchedule;
 }
 
-export enum UserStatus {
-  available = "available",
-  busy = "busy",
+export enum DefaultUserStatus {
+  online = "Online",
+  busy = "Busy",
+}
```

Comment:
I don't think that it's actually needed.. Looks like we need that only for `USER_STATUSES` const. 
But open to your thoughts about that.

Reply:
I'd say keep it for now, since these two are the default values and should always be rendered if the venue user statuses are enabled. Even if you don't have custom ones, they are also hardcoded and not removable by design.